### ISSUE TITLE
Sanitize input (prevent sh/sql injections) 

### ISSUE THREAD

**Issue by NikolayS** [NikolayS/postgrest-google-translate#3]

**Comment by ruslantalpa** [comment]
maybe this will help
https://github.com/pramsey/pgsql-http
plus it will not start a whole new os process to make http request

**Comment by NikolayS** [comment]
@ruslantalpa yes, I thought about that module, but at first stage, I wanted to keep things as simple as possible (pgsql-http requires compilation, while plsh os is already available as a package on many systems).
Anyway, it's worth to think more in this direction, thanks.
